### ISSUE TITLE
Update 

### ISSUE THREAD

**Issue by vincentsarago** [vincentsarago/landsatgif#3]
This code need a long overdue update. 

### To Do!
- per #2 moving from `convert` to a pure python version
- make it a proper python module
- use https://github.com/sat-utils/sat-search for Landsat/Sentinel search 
- make it Landsat Multi Collection Friendly
- Move from gdal to Rasterio

**Comment by fredliporace** [comment]
The move from gdal to rasterio may also solve the [issue](https://github.com/fredliporace/sentinelgif/issues/1) I found while adapting the script to work with Sentinel images.

**Comment by fredliporace** [comment]
I'm interested to dig on the available searching capabilities, could extend the search to use sat-search.

**Comment by vincentsarago** [comment]
@fredliporace 
> The move from gdal to rasterio may also solve the issue I found while adapting the script to work with Sentinel images.

I don't think this will solve it, I think it might be a driver problem!

**Comment by vincentsarago** [comment]
@fredliporace 
> I'm interested to dig on the available searching capabilities, could extend the search to use sat-search.

Yeah in fact I head back to direct api-cal: https://github.com/vincentsarago/landsatgif/blob/update/landsatgif/utils.py#L63-L87

The branch `update` still need so work (FYI, I'm using a local version of rasterio with some bug fix that we are going to push soon to master) 

everything should be done by tomorrow :-)

**Comment by fredliporace** [comment]
@vincentsarago 

>> The move from gdal to rasterio may also solve the issue I found while adapting the script to work with Sentinel images.

> I don't think this will solve it, I think it might be a driver problem!

You are right, the messages disappeared after upgrading to GDAL 2.2.0
